src/request.js

```diff
@@ -193,6 +193,13 @@ const Request = stampit().compose(ConfigMixin, Logger)
         })
         .catch((err) => {
           if (err.status && err.response) {
+
+            if(err.status === 429) {
+              const delay = _.toInteger(err.response.headers['retry-after']) * 1000;
```

Comment:
@23doors ok, I thought the `retry-after` header doesn't return values smaller than a second ;)